### ISSUE TITLE
[Feature request] Support for parsing of multiple arguments in callees

### ISSUE THREAD

**Issue by NatanCieplinski** [francoismassart/eslint-plugin-tailwindcss#15]
**Is your feature request related to a problem? Please describe.**
In our project we use a lot the [classNames](https://github.com/JedWatson/classnames) utility to merge the styles coming from different places. Configuring the `classnames` function as a callee makes the rule work only if the function has one arguments. 
For example this will be correctly linted:
```jsx
<div className={classnames(`flex overflow-hidden flex-col items-stretch mx-auto rounded-lg border border-grey-3`)}>
```
While this will not:
```jsx
<div className={classnames(
   styles.background, 
   `flex overflow-hidden flex-col items-stretch mx-auto rounded-lg border border-grey-3`
)}>
```

**Describe the solution you'd like**
I would like to be able to pass `n` arguments to the callee and if one of those is a string containing tailwind classes, to be parsed by the rule 

**Additional context**
This is the current config
```
"tailwindcss/classnames-order": [2, {
      "callees": ["classnames"]
    }]
```

**Comment by francoismassart** [comment]
FIX released in v1.8.0

https://github.com/francoismassart/eslint-plugin-tailwindcss/releases/tag/v1.8.0
